README.md

```diff
@@ -38,27 +38,33 @@ These are the key requirements which the generator must fulfil.
         * ID of the event
     @param Dictionary $eventAttributes
         * Attributes of the event. Key is name of attribute, value is function generating the value
+    @param Dictionary $eventAttributes
+        * Attributes of the event. Key is name of attribute, value is function generating the value
     @return Array[Event]
         * Array of multiple events if it is a template, or returning a single element array
 **/
 constructor(id, eventAttributes)
 ```
-Atomic object which gets creates the customer history. `eventAttributes` is a dictionary which describes the event. It can have following specifications:
+Atomic object which gets creates the customer history. `eventAttributes` is a dictionary which describes the event., `resources` is a dictionary that handles global resources that can be used when generating event attributes. It can have following specifications:
```

Comment:
I would change phrasing `global` resources to `resources within the scope of this Event and Session`